internal/command/views/modules.go

```diff
@@ -38,9 +39,34 @@ type ModulesHuman struct {
 var _ Modules = (*ModulesHuman)(nil)
 
 func (v *ModulesHuman) Display(manifest moduleref.Manifest) int {
+	if len(manifest.Records) == 0 {
+		v.view.streams.Println("No modules found in configuration.")
+		return 0
+	}
+	printRoot := treeprint.New()
+	populateTreeNode(printRoot, &moduleref.Record{
+		Children: manifest.Records,
+	})
+
+	v.view.streams.Println(fmt.Sprintf("\nModules declared by configuration:\n%s", printRoot.String()))
 	return 0
 }
 
+func populateTreeNode(tree treeprint.Tree, node *moduleref.Record) {
+	for _, childNode := range node.Children {
+		item := fmt.Sprintf("\"%s\"[%s]", childNode.Key, childNode.Source.String())
```

Comment:
I'd have expected `%q` instead of `\"%s\"` but I'm more curious if there is a reason for quoting the module key in the _human_ output at all? 🤔